docs/source/onnx.rst

```diff
@@ -7,7 +7,7 @@ Example: End-to-end AlexNet from PyTorch to Caffe2
 
 Here is a simple script which exports a pretrained AlexNet as defined in
 torchvision into ONNX.  It runs a single round of inference and then
-saves the resulting traced model to ``alexnet.proto``::
+saves the resulting traced model to ``alexnet.onnx``::
 
     from torch.autograd import Variable
```

Comment:
not for this PR, but we need to update these `Variable` and `.cuda()` to 0.4 standards sometime.